src/PhotoNim.nim

```diff
@@ -57,8 +62,24 @@ proc pfm2png(fileIn, fileOut: string, alpha, gamma: float32) =
     echo fmt"Successfully converted {fileIn} to {fileOut}"
 
 
+proc col_pix(im_tr: ImageTracer, ray: Ray, scenary: World, row, col: int): Color = 
+    # Procedure to decide pixel color (it could be useful to check if scenary len is non zero)
```

Comment:
This must be implemented before the merge: if the len of the array containing the shapes is 0, then the color should be set to the background color of the world.